README.md

```diff
@@ -62,10 +62,11 @@ for that dataset. The following is an example of a dataset object:
 
 ```json
 {
+  "eventsTable": "indonesia_v20181029",
   "pipeline": "pipe_indonesia_production_v20181029",
-  "bigquery": {"
-    fishingTable": "world-fishing-827.pipe_indonesia_production_v20181029.published_events_fishing"
-  }
+  "tracksTable": "indonesia_v20181029_tracks",
+  "vesselIndex": "andres"-test-indonesia,
```

Comment:
@andres-arana I would also add a comment for each datasource explaining which pipeline generates it, which format, and other specific thngs needed to create them.

eg:

* `tracksTable`
  
  * It is a postgres db that contantaoins the tracks with a X format, it is created and published by dag.
* `vesselIndex`
  
  * It is a Elastic search index, it is created by dag

Reply:
I'll add the docs to a shared wiki page, since we will need those docs on the `api-vessels` project as well.